client/my-sites/customer-home/controller.jsx

```diff
@@ -20,9 +22,20 @@ export default async function ( context, next ) {
 export function maybeRedirect( context, next ) {
 	const state = context.store.getState();
 	const slug = getSelectedSiteSlug( state );
+
 	if ( ! canCurrentUserUseCustomerHome( state ) ) {
 		page.redirect( `/stats/day/${ slug }` );
 		return;
 	}
+
+	const launchpadTasksComplete = false;
+	const launchpadFlow = 'newsletter';
+
+	if ( ! launchpadTasksComplete && isEnabled( 'signup/launchpad' ) ) {
```

Comment:
I imagine once we have the setting to check that this logic might be inversed.  That is, something where a falsy value would mean we dont show launchpad (like `isLaunchpadActive`), so old sites without launchpad or any setting added yet don't get mixed in at some point in the future.

Reply:
Implemented references to actual site options here [72e69f9](https://github.com/Automattic/wp-calypso/pull/66613/commits/72e69f9731b80cc5da649142e3a257a212d50cdd)